search-server/spacewalk-search/src/java/com/redhat/satellite/search/rpc/XmlRpcInvoker.java

```diff
@@ -55,14 +57,18 @@ public XmlRpcInvoker(XmlRpcServer xmlrpcServer) {
      */
     public void handle(Request request, Response response) {
         String uri = request.getURI();
-        log.info(uri);
+        InetAddress ip = request.getInetAddress();
+
         try {
             if (!uri.startsWith("/RPC2")) {
+                String url = StringUtil.htmlifyText(uri);
+
+                log.info("Invalid request from {} to {}", ip, uri);
```

Comment:
One small detail I guess
```suggestion
                log.info("Invalid request from {} to {}", ip, url);
```

Reply:
ok, resolving